Source/Engine/Level/Actor.h

```diff
@@ -382,6 +382,15 @@ API_CLASS(Abstract) class FLAXENGINE_API Actor : public SceneObject
         return _isActiveInHierarchy != 0;
     }
 
+    /// <summary>
+    /// Gets value indicating if actor is in a scene.
+    /// </summary>
+    API_PROPERTY(Attributes="HideInEditor, NoSerialize")
+    FORCE_INLINE bool IsInScene() const
```

Comment:
I would use name `HasScene` to match naming convention in Actor: `HasParent`/`HasTag`/`HasPrefabLink`/..